PowerShellTools.HostService/ServiceManagement/Debugging/PowerShellDebuggingService.cs

```diff
@@ -996,7 +1134,8 @@ public void SetOption(PowerShellRawHostOptions options)
         }
 
         /// <summary>
-        /// Returns the connection info for the current runspace
+        /// Returns the connection info for the current runspace based on certain characteristics of the runspace. Note: the criteria for remote debug vs remote session
+        /// is not always correct and should be treated carefully.
         /// </summary>
         public DebugScenario GetDebugScenario()

```

Comment:
We do want to expose it. It is used a few places outside of the host. (example: when detaching so we know which public detach service to call)